src/components/MlTemporalController/MlTemporalController.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import MlTemporalController, { MlTemporalControllerProps } from './MlTemporalController';
+import mapContextDecorator from '../../decorators/MapContextDecorator';
+import { Box, Typography } from '@mui/material';
+import TopToolbar from '../../ui_components/TopToolbar';
+import african_independency from './assets/african_independency.json';
```

Comment:
We need license info & visible attribution in the demo (can be passed to MlGeoJsonLayer options.source.attribution).

Reply:
das funktioniert immer noch nicht... Ich lege gerade einen Ticket dafür und schaue, ob ich selber repariert kriege.

Reply:
https://easyredmine.wheregroup.com/issues/7305